src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/FigaroValidationServiceConstants.java

```diff
@@ -11,10 +11,9 @@
 	
 	private static final String VALIDATION_CORRELATION_ID = "correlationId";
 	
-	// Response code types
-	public static final int VALIDATION_SERVICE_FAILURE_CD = 0;
-	public static final int VALIDATION_SERVICE_SUCCESS_CD = 1;
-	
+	// Response code types; anything >= 0 is good, anything < 0 is bad. 
+	public static final int VALIDATION_SERVICE_FAILURE_CD = -1;
```

Comment:
so we could throw -10 and the effect is the exact same?

Reply:
Yup. All values originate from the Database. The caller (KOFAX) knows to watch for anything below 0 and raise and issue.